shared/constants.js

```diff
@@ -60,12 +60,12 @@ module.exports = {
         },
         cordova: {
             checkCmd: 'cordova -v',
-            pluginRepoUri: 'https://github.com/forcedotcom/SalesforceMobileSDK-CordovaPlugin#dev',    // dev
+            pluginRepoUri: 'https://github.com/wmathurin/SalesforceMobileSDK-CordovaPlugin#dev',    // dev
```

Comment:
Will revert this line once everything passes and this [PR](https://github.com/forcedotcom/SalesforceMobileSDK-CordovaPlugin/pull/624) is merged.